conda-recipe/meta.yaml

```diff
@@ -43,8 +43,6 @@ test:
         - scikit-learn
         - mpich # [osx]
         - impi_rt # [not osx]
-    imports:
-        - daal4py
```

Comment:
What this would change? per my understanding current setup works in same way?

Reply:
No, current setup worked (before merge of this PR) with check of importing before running other tests.
https://github.com/IntelPython/daal4py/blob/7334aeb6cf7cc53c25aaceb2f62c5041a6c4969a/conda-recipe/meta.yaml#L46
It can be significant if you will get fail in tests with MPI because the errors from MPI are often not informative so we will not understand that we simple failing during import.